lib/common/exceptions/service_unavailable.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Common
+  module Exceptions
+    class ServiceUnavailable < ServiceError
+    end
```

Comment:
Don't we already have this handled? Why do we need a new error class? If a backend service is unavailable we use Internal/ServiceError.

The only error Backend services should ever raise is BackendServiceException. From there it should either become some kind of InternalServerError or something else that is application specific. I don't think you need a new error class here. And if you're trying to suppress or change how Sentry reports, it should be done outside of client, in a processor or application_controller (for more consistency with how we do it elsewhere)... Also, not entirely clear why this needs to be a global decision. For Rx/SM, we're content with 503s being errors vs warnings I think.